internal/connect/command_offset_describe.go

```diff
@@ -0,0 +1,180 @@
+package connect
+
+import (
+	"encoding/json"
+
+	connectv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect/v1"
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafka"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type serializedOffsetConnectorOut struct {
+	Id       string                                      `json:"id" yaml:"id"`
```

Comment:
Any reason we're not supporting `human` output here too?

Reply:
(Also, it's more appropriate to use our custom `serialized` tag here, but that might require formatting the offsets as a string)